cmd/client-s3.go

```diff
@@ -219,6 +220,12 @@ func getCredentialsChainForConfig(config *Config, transport http.RoundTripper) (
 		// set AWS_WEB_IDENTITY_TOKEN_FILE is MC_WEB_IDENTITY_TOKEN_FILE is set
 		if val := env.Get("MC_WEB_IDENTITY_TOKEN_FILE", ""); val != "" {
 			os.Setenv("AWS_WEB_IDENTITY_TOKEN_FILE", val)
+			if val := env.Get("MC_ROLE_ARN", ""); val != "" {
```

Comment:
If I am not mistaken, a better way is to make mc looks for 'alias', then env vars, then local FS; but this seems to be out of scope of this PR.

Reply:
Alias is frozen @vadmeste lets not make wide changes for minor requirements, this is good enough for most people.